src/declarations/signer/signer.factory.did.js

```diff
@@ -126,7 +126,18 @@ export const idlFactory = ({ IDL }) => {
 		outputs: IDL.Vec(BtcTxOutput)
 	});
 	const SendBtcResponse = IDL.Record({ txid: IDL.Text });
+	const BuildP2wpkhTxError = IDL.Variant({
+		NotEnoughFunds: IDL.Record({
+			available: IDL.Nat64,
+			required: IDL.Nat64
+		}),
+		WrongBitcoinNetwork: IDL.Null,
+		NotP2WPKHSourceAddress: IDL.Null,
+		InvalidDestinationAddress: GetAddressResponse,
+		InvalidSourceAddress: GetAddressResponse
+	});
 	const SendBtcError = IDL.Variant({
+		BuildP2wpkhError: BuildP2wpkhTxError,
 		InternalError: IDL.Record({ msg: IDL.Text }),
 		PaymentError: PaymentError
```

Comment:
It is odd that the new entry is the first variant.  Normally variants are numbered 0, 1, 2... and adding to the beginning will break anything that uses numeric indices to encode variants.